source/Pages/best_practices.rst

```diff
@@ -0,0 +1,160 @@
+
+
+.. _running-many-jobs:
+
+Running a large amount of jobs 
+==============================
+
+When running a large amount of jobs it can be difficult to keep track of the state of these jobs or resume failed tasks that were prematurely canceled due to time limit. Another challenge when designing high-throughput workflows that execute a specific application for many different parameter combinations, is reducing the large scheduling overhead and waiting times in the queue. 
+
+
+An overview of the features and capabilities of some of the options for running a large amount of jobs on Spider is presented below.
+
+==============================================   ==============   ==============   =============
+Feature                                          Array jobs       Picas            Snakemake
+==============================================   ==============   ==============   =============  
+High speed & Low load on the system              No               Yes              Somewhat  
+Scales to hundreds, thousands of jobs and more   No               Yes              Somewhat
+Available outside Spider                         No               Yes              No
+Easy setup                                       Yes              Somewhat         Somewhat    
+Handles easily dependencies between tasks        No               Somewhat         Yes
+Error recovery                                   No               Yes              No
+==============================================   ==============   ==============   =============
+
+
+Which practice for running a large amount of jobs should I use?
+---------------------------------------------------------------
+
+The first option to check when running a large amount of jobs is whether the software you're using comes with a built-in option for managing your workloads on a Slurm-based cluster. Alternatively, an easy way to submit several independent jobs with one command is the use of Array jobs. Array jobs, however do not scale well for more than a few hudrends of jobs. In this case, you can use external tools for managing your workloads, such as PiCaS or Snakemake.
+
+
+PiCaS works as a queue, providing a mechanism to step through the work one task at a time. It is also a pilot job system, indicating that the client communicates with the PiCaS server to fetch work, instead of having that work specified in a job (or similar) file.  As every application needs different parameters, PiCaS has a flexible data structure that allows users to save different types of data. PiCaS can handle thousands or millions of tasks, it has an easy query mechanism to search among your tasks and is accessible from any platform via a Restful HTTP API. Here you can find instructions for using :ref:`PiCaS <picas-on-spider>`.
```

Comment:
```suggestion
PiCaS works as a queue, providing a mechanism to step through the work one task at a time. It is also a `pilot job <https://doc.grid.surfsara.nl/en/latest/Pages/Practices/pilot_jobs.html>`_  system, indicating that the client communicates with the PiCaS server to fetch work, instead of having that work specified in a job (or similar) file.  As every application needs different parameters, PiCaS has a flexible data structure that allows users to save different types of data. PiCaS can handle thousands or millions of tasks, it has an easy query mechanism to search among your tasks and is accessible from any platform via a Restful HTTP API. Here you can find instructions for using :ref:`PiCaS <picas-on-spider>`.
```